cppcache/src/StreamDataInput.cpp

```diff
@@ -60,9 +58,7 @@ void StreamDataInput::readDataIfNotAvailable(size_t size) {
             .c_str(),
         std::chrono::duration_cast<std::chrono::milliseconds>(timeSpent)
             .count(),
-        std::chrono::duration_cast<std::chrono::milliseconds>(
-            remainingTimeBeforeTimeout_)
-            .count());
+        remainingTimeBeforeTimeout_.count());
 
     if (remainingTimeBeforeTimeout_ <= std::chrono::milliseconds ::zero()) {
```

Comment:
Surprised the formatter didn't catch this but what's with the space between milliseconds and zero?

Reply:
Weird...